willyserge/broadcaster
# #169937549 admin should get a list of all incidents

#### What does this PR do?
  an endpoint to get a list of all red-flags
#### Description of Task to be completed?
working endpoint
`/api/v1/incidents/admin`
#### How should this be manually tested?
- run `npm install` to install all dependencies
- run `npm start `and to start server
- check your `port number` the server is running on
- go to `postman` and make a` PATCH request` to `http://localhost: port number /api/v1/incidents/admin`
#### What are the relevant pivotal tracker stories?
#169937549
